markdown/preamble/version-history.md

```diff
@@ -1,7 +1,30 @@
 Version history
 ===============
 
-Version 1.20 (2016-04-25 / DRAFT IN PROGRESS):
+Version 1.30 (2019-01-14):
+
+-   This specification is now published under the terms of the W3C Final Specification Agreement (FSA) ([#38](https://github.com/w3c/smufl/issues/38))
```

Comment:
As in the HTML, this should refer to the W3C Community Final Specification Agreement.